src/install-elixir.sh

```diff
@@ -9,12 +9,8 @@ FILE_INPUT="${VSN}.zip"
 FILE_OUTPUT=elixir.zip
 DIR_FOR_BIN=.setup-beam/elixir
 
-rm -f "${FILE_OUTPUT}"
```

Comment:
Removed what was unnecessary, even if old code. It makes for easier maintenance.